scripts/local-cluster.sh

```diff
@@ -0,0 +1,425 @@
+      }
+    },
+    "nodeLocalDns": {
+      "customConfig": ("'"$(print_coredns_config)"'")
+    },
+    "networkPolicies": {
+      "global": {
+        "objectStorage": {
+          "ips": [ "0.0.0.0/0" ],
+          "ports": [ 9000 ]
+        },
+        "scIngress": {
+          "ips": [ "0.0.0.0/0" ]
+        },
+        "wcIngress": {
+          "ips": [ "0.0.0.0/0" ]
+        },
+        "externalLoadBalancer": false,
+        "ingressUsingHostNetwork": false,
+        "trivy": { "ips": [ "0.0.0.0/0"] }
+      },
+      "certManager": {
+        "letsencrypt": { "ips": [ "0.0.0.0/0" ] }
+      },
+      "coredns": {
+        "externalDns": { "ips": [ "0.0.0.0/0" ] }
+      },
+      "falco": {
+        "plugins": { "ips": [ "0.0.0.0/0" ] }
+      },
+      "alertmanager": {
+        "alertReceivers": { "ips": [ "0.0.0.0/0" ] }
+      }
+    }
+  }' "${CK8S_CONFIG_PATH}/common-config.yaml"
+
+  yq4 -Pi 'with(select(. == null); . = {}) | . *= {
+    "alerts": {
+      "opsGenieHeartbeat": {
+        "enabled": false,
+        "name": "not-used"
+      }
+    },
+    "grafana": {
+      "ops": {
+        "oidc": {
+          "allowedDomains": [ "example.com" ]
+        }
+      },
+      "user": {
+        "oidc": {
+          "allowedDomains": [ "example.com" ]
+        }
+      }
+    },
+    "harbor": {
+      "persistence": {
+        "disableRedirect": true,
+        "type": "objectStorage"
+      },
+      "oidc": {
+        "adminGroupName": "admin"
+      }
+    },
+    "opensearch": {
+      "extraRoleMappings": [{
+        "mapping_name": "all_access",
+        "definition": {
+          "users": [ "admin@example.com" ]
+        }
+      }]
+    },
+    "networkPolicies": {
+      "dex": { "connectors": { "ips": [ "0.0.0.0/0" ] } },
+      "harbor": {
+        "registries": { "ips": [ "0.0.0.0/0" ] },
+        "jobservice": { "ips": [ "0.0.0.0/0" ] },
+        "trivy": { "ips": [ "0.0.0.0/0" ] }
+      },
+      "monitoring": {
+        "grafana": { "externalDashboardProvider": { "ips": [ "0.0.0.0/0" ] } }
+      },
+      "opensearch": { "plugins": { "ips": [ "0.0.0.0/0" ] } }
+    }
+  }' "${CK8S_CONFIG_PATH}/sc-config.yaml"
+
+
+  yq4 -Pi 'with(select(. == null); . = {}) | . *= {
+    "user": {
+      "namespaces": [ "production", "staging" ],
+      "adminUsers": [ "admin@example.com" ],
+      "adminGroups": [ "admin" ]
+    }
+  }' "${CK8S_CONFIG_PATH}/wc-config.yaml"
+
+
+  if ! [[ -f "${CK8S_CONFIG_PATH}/defaults/common-config.yaml" ]]; then
+    mkdir -p "${CK8S_CONFIG_PATH}/defaults"
+    touch "${CK8S_CONFIG_PATH}/defaults/common-config.yaml"
+    yq4 -Pi '. = {
+      "global": {
+          "ck8sCloudProvider": "none",
+          "ck8sEnvironmentName": "'"${name}"'",
+          "ck8sFlavor": "'"${flavor}"'"
+      }
+    }' "${CK8S_CONFIG_PATH}/defaults/common-config.yaml"
+  fi
+
+  "${ROOT}/bin/ck8s" init both
+}
+
+create() {
+  if [[ -z "${1:-}" ]]; then
+    log.usage
+  fi
+
+  local config
```

Comment:
Just to make it a bit more readable imo
```suggestion
  local config cluster
  
  cluster="${1}"
```